includes/functions/plugins/article-export.php

```diff
@@ -349,12 +351,15 @@ function build_file_sets() {
 	function get_xml_file( $article ) {
 		$xml = $this->get_xml( $article );
 
-		$file_name = apply_filters( __NAMESPACE__ . '\xml_filename', $xml->root_element->getAttribute( 'title' ), $article, $xml );
+		$file_name  = apply_filters( __NAMESPACE__ . '\xml_filename', $xml->root_element->getAttribute( 'title' ), $article, $xml );
 		$file_name .= '.xml';
 
 		$file_contents = $xml->xml_document->saveXML();
 
-		return new File( $file_contents, apply_filters( __NAMESPACE__ . '\xml_full_filename', $file_name, $article ) );
+		$fileset   = array();
+		$fileset[] = new File( $file_contents, apply_filters( __NAMESPACE__ . '\xml_full_filename', $file_name, $article ) );
+
+		return $fileset;
```

Comment:
We are now returning an `array` instead of a `File` object. Assuming that's what we want, we'll want to update the docblock on this function to reflect that

Reply:
@dkotter This was taken care of here: https://github.com/10up/eight-day-week/blob/950ced938e388276dad7db0cfb1fb8d92e862a0e/includes/functions/plugins/article-export.php#L368

Reply:
Updated